lib/commander/help_formatters/terminal/help.erb

```diff
@@ -6,10 +9,11 @@
 
     <%= Commander::HelpFormatter.indent 4, program(:description) %>
 
-  <%= $terminal.color "COMMANDS", :bold %>:
+  <%= "%s: %s" % [$terminal.color("COMMANDS", :bold), default_command.nil? ? '' : '(* default)'] %>
 <% for name, command in @commands.sort -%>
 	<% unless alias? name %>
-    <%= "%-#{max_command_length}s %s" % [command.name, command.summary || command.description] -%>
+    <% summary_prefix = (!default_command.nil? && command.name == default_command.name)? '* ' : '  ' -%>
```

Comment:
We may be missing a spec for this, but does this increase the indentation level of existing help output in cases where there's _not_ a default command? It would be nice to avoid changing the current output style.